CHANGES.rst

```diff
@@ -1,8 +1,20 @@
 Changelog
 =========
 
-3.0.1 (unreleased)
-------------------
+3.1 (unreleased)
+----------------
+
+- Plone 5 + Plone 4 compatibility.
+  [thet]
+
+- Extend the ``uninstall`` profile with more de-registrations.
+  [thet]
+
+- Add a ``available`` method to the transform, which can be overridden to e.g. allow the transform also for non-anonymous users.
```

Comment:
I would prefer you to help me reviewing #35; I prepared it for @gforcada, but it seems he was not very excited about it ;-)

Reply:
ok, there is another PR - didn't check for it.
an option makes totally sense - maybe in combination with the ``available`` method done by me. i could think of other use cases to disable the transform - e.g. on a per content type basis.

Reply:
Reviewed!

Reply:
Actually - I guess I'll remove the commit with factoring out the ``available`` method.